CentumFramework/GoogleSheets/Models/MergeType.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+using CentumFramework.Utility.Attributes;
+using CentumFramework.Utility.Extensions;
+
+namespace CentumFramework.GoogleSheets.Models
+{
+    public enum MergeType
+    {
+        [StringValue("MERGE_ALL")]
+        All,
+        [StringValue("MERGE_COLUMNS")]
+        Columns,
+        [StringValue("MERGE_ROWS")]
+        Rows
```

Comment:
```suggestion
        All = 1,
        [StringValue("MERGE_COLUMNS")]
        Columns = 2,
        [StringValue("MERGE_ROWS")]
        Rows = 3
```

Если не указывать явно, то енам начнется с 0 => в любой непонятной ситуации, когда енам проинициализируется дефолтным для структур значением (0), ты будешь думать, что все ок и значение All. Хотя на самом деле там может вообще полпроекта умерло